src/plugins/intel_cpu/src/transformations/snippets/x64/pass/lowered/brgemm_blocking.cpp

```diff
@@ -62,34 +62,135 @@ bool BrgemmBlocking::run(snippets::lowered::LinearIR& linear_ir) {
 
     bool modified = false;
     for (auto expr_it = linear_ir.begin(); expr_it != linear_ir.end(); expr_it++) {
-        const auto& expr = *expr_it;
-        const auto brgemm = ov::as_type_ptr<ov::intel_cpu::BrgemmCPU>(expr->get_node());
-        if (!brgemm || blocking_loop_exists(expr, brgemm))
+        const auto& brgemm_expr = *expr_it;
+        const auto brgemm = ov::as_type_ptr<ov::intel_cpu::BrgemmCPU>(brgemm_expr->get_node());
```

Comment:
I know that this is a draft PR, but still...Please note: we can't refer to cpu opset from the common part